dev/src/user.clj

```diff
@@ -1,4 +1,9 @@
-(ns user)
+(ns user
+  (:require
+   [metabase.test-runner.assert-exprs]))
+
+;; make sure stuff like `schema=` and what not are loaded
+(comment metabase.test-runner.assert-exprs/keep-me)
```

Comment:
Not directly related but I did get into a situation where I started a REPL and started running tests in `metabase.async.api-response-test` and ran into "no method for `schema=`" or whatever errors so adding this seems like a good idea